### PR TITLE
Update OCP levels to 4.17 for ci tests

### PR DESCRIPTION
**Pull request by josephca**
Upgrade the OCP levels from 4.14/4.15/4.16 to 4.15/4.16/4.17
https://issues.redhat.com/browse/RHIDP-4960

### CODE REVIEW

**Comment by josephca** [comment]
Hi @subhashkhileri, please review. Thanks!

File: ci-operator/config/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main.yaml [hunk 1]
```diff
@@ -35,18 +35,18 @@ tests:
   steps:
     test:
     - ref: janus-idp-backstage-showcase-nightly
-- as: e2e-tests-nightly-ocp-v4-15
+- as: e2e-tests-nightly-ocp-v4-16
```
Comment (subhashkhileri) [inline ci-operator/config/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main.yaml:38]: @josephca I guess we can keep the ocp 15 as it is and update the 14 to 16

Comment (josephca) [reply ci-operator/config/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main.yaml:38]: > @josephca I guess we can keep the ocp 15 as it is and update the 14 to 16

@subhashkhileri correct. The change would change the execution order of OCP 16 first then 15 which doesn't really matter, correct? :)

Comment (subhashkhileri) [reply ci-operator/config/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main.yaml:38]: right

**Comment by josephca** [comment]
/pj-rehearse pull-ci-janus-idp-backstage-showcase-main-e2e-tests

**Comment by josephca** [comment]
/pj-rehearse periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-ocp-v4-16

**Pull request merged**